### PR TITLE
V2.1.0

### PR DESCRIPTION
**Pull request by mberacochea**

### CODE REVIEW

File: containers/gtdb-tk/Dockerfile
Comment (Ales-ibt) [inline containers/gtdb-tk/Dockerfile:1]: Not sure why do we need this Dockerfile if Nextflow is using the container from quay

File: containers/interproscan/Dockerfile
Comment (Ales-ibt) [inline containers/interproscan/Dockerfile:1]: Same here. Do we need the Docker file if Nextflow uses the container from quay?

**Review by Ales-ibt — CHANGES_REQUESTED** [review]
The code for updating databases and tools version looks OK to me. I just was wondering, why do we have Dockerfiles for some tools if Nextflow is using the containers from quay?

File: containers/interproscan/Dockerfile
Comment (tgurbich) [reply containers/interproscan/Dockerfile:1]: The container nextflow uses was uploaded to quay by us and the Dockerfile is the one from which it was made so we keep them.

Comment (mberacochea) [reply containers/interproscan/Dockerfile:1]: Quay is just the hosting platform, we have the images there:
```
container 'quay.io/microbiome-informatics/genomes-pipeline.gtdb-tk:v2.3.0'
```
that is the one we pushed for GTDBtk.

**Pull request merged**